packages/shared/src/domain/contact/phone.ts

```diff
@@ -3,6 +3,11 @@ import { stripNonNumericChars } from "../../common/string";
 export const phoneLength = 10;
 export const examplePhoneNumber = "1231231234";
 
+export function isPhoneNumber(phone: string): boolean {
+  const numericChars = phone.replace(/\D/g, "");
+  return numericChars.length >= 10 && !phone.includes("@");
```

Comment:
`!phone.includes("@");` won't do anything -- you've remove non-numeric characters...